CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+
+## Web Component Requirements
+
+Please ensure that all Web Components contributed meet the following guidelines:
+
+* Please ensure your component javascript is written in ES6 and follows our eslint standards within the provided `.eslintrc` file. 
+* Provide a test page for your component which demonstrates its UI features and functional behaviors. This should also demonstrate how it is used alongside Patternfly CSS and if additional CSS is needed.
+   * Ideally, all the css that is needed for the components in Patternfly Web Components would be defined in the core css [Patternfly](https://github.com/patternfly/patternfly) repo. However, there may be cases where additional css is needed to complete the Patternfly Web Component contribution. In the case where the css is specific to the component implementation in the Patternfly Web Component repo, then the css would remain in the repo. In the case where the component does not exist yet in the core css repo and css is needed in the Patternfly Web Component repo to style the component, then the css would only remain in the Patternfly Web Component repo until the component is added to the core css repo. Once the css is available as part of core css repo, it will be removed from the Patternfly Web Component repo.
+
+* Provide unit tests for your component in a corresponding `.spec.js` file. These should demonstrate the component's API and test its underyling functional pieices individually. 
+* Ensure the component's rendered design meets all [PatternFly design standards](https://github.com/patternfly/patternfly-design) for this component.
+**Note:** If your component does not yet have PatternFly design documentation, the PatternFly Web Component design team will first confirm that the pattern passes the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) and then start the process for generating design documentation.
+
+## Code Contribution Guidelines
+
+Adhering to the following process is the best way to get your work included in the project:
+
+1. [Fork](https://help.github.com/fork-a-repo/) the project, clone your fork, and configure the remotes:
+
+  ```bash
+  # Clone your fork of the repo into the current directory
+  git clone https://github.com/<your-username>/patternfly-webcomponents.git
+  # Navigate to the newly cloned directory
+  cd patternfly-webcomponents
+  # Assign the original repo to a remote called "upstream"
+  git remote add upstream https://github.com/patternfly-webcomponents/patternfly-webcomponents.git
+  ```
+2. Create a branch:
+
+  ```text
+  $ git checkout -b my-branch -t origin/master
+  ```
+
+3. Develop your component. After development is complete, ensure tests and lint standards pass.
+
+  ```text
+  $ gulp lint
+  $ gulp test
```

Comment:
submitted a PR #49 that references ```npm``` scripts. It'd be nice if continued the npm script aliasing references instead